assets/_scss/_navigation.scss

```diff
@@ -422,4 +422,4 @@
       display: block;
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Just saw `insert_final_newline = false` in your `.editorconfig` file and had to do a double take on my editor. But it's enabled. YOLO